File: app/src/main/java/me/bilalhaider/moviedatabase/MovieDatabaseApplication.java
package me.bilalhaider.moviedatabase;

import android.app.Application;

import me.bilalhaider.moviedatabase.data.network.API;
import me.bilalhaider.moviedatabase.data.network.ApiService;

public class MovieDatabaseApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
    }

    public static API getApiService() {
        return ApiService.createService(API.class);
    }
}

File: app-android/src/main/java/me/bilalhaider/moviedatabase/MovieDatabaseApplication.kt
package me.bilalhaider.moviedatabase

import android.app.Application
import dagger.hilt.android.HiltAndroidApp
import me.bilalhaider.moviedatabase.data.network.API
import me.bilalhaider.moviedatabase.data.network.ApiService

@HiltAndroidApp
internal class MovieDatabaseApplication : Application() {
    override fun onCreate() {
        super.onCreate()
    }

    companion object {
        @JvmStatic
        val apiService: API
            get() = ApiService.createService(
                API::class.java)
    }
}